policyengine_canada/variables/gov/provinces/qc/tax/income/credits/work_premium/qc_adapted_work_premium_single_amount.py

```diff
@@ -0,0 +1,36 @@
+from policyengine_canada.model_api import *
+
+
+class qc_adapted_work_premium_single_amount(Variable):
+    value_type = float
+    entity = Household
+    label = "Quebec adapted work premium tax credit for singles filers"
+    definition_period = YEAR
+    defined_for = "qc_adapted_work_premium_eligibility"
+
+    def formula(household, period, parameters):
+        p = parameters(
+            period
+        ).gov.provinces.qc.tax.income.credits.work_premium.adapted_work_premium
+
+        # family situation
+        single = ~household("is_married", period)
+        has_child = household("count_children", period) > 0
+
+        income = household("adjusted_family_net_income", period)
+
+        # work income eligibility
+        work_income_eligible = (
+            household("family_working_income", period) > p.work_income_limit
+        )
+
+        eligible = work_income_eligible & single
```

Comment:
we dont need to check for the eligibility if we use qc_adapted_work_premium_eligibility